src/main/target/common_pre.h

```diff
@@ -65,7 +65,6 @@
 #define USE_TIMER_MGMT
 #define USE_PERSISTENT_OBJECTS
 #define USE_CUSTOM_DEFAULTS_ADDRESS
-#define USE_SPI_TRANSACTION
```

Comment:
Thias has to stay on, or SPI transaction support has to be enabled by default.

Reply:
Please explain. This API does away with all that.